test/performance/specs/site-editor.spec.js

```diff
@@ -295,47 +295,22 @@ test.describe( 'Site Editor Performance', () => {
 				} );
 				await editor.openDocumentSettingsSidebar();
 
-				/*
-				 * https://github.com/WordPress/gutenberg/pull/55091 updated the HTML by
-				 * removing the replace template button in sidebar-edit-mode/template-panel/replace-template-button.js
-				 * with a "transform into" list. https://github.com/WordPress/gutenberg/pull/59259 made these tests
-				 * compatible with the new UI, however, the performance tests compare previous versions of the UI.
-				 *
-				 * The following code is a workaround to test the performance of the new UI.
-				 * `actionsButtonElement` is used to check if the old UI is present.
-				 * If there is a Replace template button (old UI), click it, otherwise, click the "transform into" button.
-				 * Once the performance tests are updated to compare compatible versions this code can be removed.
-				 */
```

Comment:
This logic isn't required after #60780.

Reply:
Why not?

1. This logic is still required for WP 6.5.
2. We should be able to run performance tests for 6.5, so we can compare.